instrumentation/play-ws/play-ws-common/library/play-ws-common-library.gradle

```diff
@@ -1,7 +1,9 @@
-apply from: "$rootDir/gradle/instrumentation.gradle"
+apply from: "$rootDir/gradle/instrumentation-library.gradle"
 
 def scalaVersion = '2.12'
 
 dependencies {
+  compileOnly project(':javaagent-api')
+  compileOnly project(':javaagent-tooling')
```

Comment:
Hmm, it's not really a "piece of library instrumentation" if it depends on javaagent artifacts... I wonder if we need another "module type" like `javaagent-shared` for those modules. (In another issue/PR though)

Reply:
Yeah, I am not happy about this as well, but don't have any better solution that does not explode the scope.

Reply:
It look like the common -> library pattern worked out ok for most artifacts. For this one how about same hack as runtime metrics?

Reply:
Got back to javaagent